app/Models/WaitingRoom.php

```diff
@@ -15,4 +15,62 @@ public function users()
     {
         return $this->belongsToMany(User::class);
     }
+
+    /*
+     * Equally splits the users of a waiting room into arrays
+     * representing competitions.
+     */
+    public function getDefaultSplit()
+    {
+        $users = $this->users;
+
+        // Get the size of the waiting room
+        $roomSize = count($users);
+
+        // Determine the amount of competitions to make
+        $numOfCompetitions = $roomSize / 300;
```

Comment:
So what happens if `$roomSize` is `305`? You get a decimal number of competitions; should you `floor` or `ceil` it? Also, do we get a room with 5-ish people?